.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "semi": false,
+  "singleQuote": false
```

Comment:
This should probably be "backtick"

Reply:
https://eslint.org/docs/rules/quotes

Reply:
Seems like prettier will not change between strings and template strings, but eslint will, that's why it's only in eslintrc :/

https://github.com/prettier/prettier/issues/54#issuecomment-283439626
https://github.com/prettier/prettier/issues/2996#issuecomment-335223208

Reply:
Ah, yes, I got confused here. Thought I was looking at the eslintrc file. 

Sadly, Prettier is ignoring them. I have a plugin for eslint that does the conversion for me that works really well. Would have preferred this to be built-in for Prettier.